.github/workflows/wasm32.yml

```diff
@@ -51,7 +51,7 @@ jobs:
         run: make crystal # TODO: Remove this after next version update
 
       - name: Build spec/wasm32_std_spec.cr
-        run: bin/crystal build spec/wasm32_std_spec.cr -o wasm32_std_spec.wasm --target wasm32-wasi --release
+        run: bin/crystal build spec/wasm32_std_spec.cr -o wasm32_std_spec.wasm --target wasm32-wasi -Duse_pcre --release
```

Comment:
The wasm lib for PCRE2 should be available since #13109. Why `-Duse_pcre`?

Reply:
I was just merging master, so this is what's in master today

Reply:
Oh, but that seems to be a mistake. Apparently 46a6ac73e7e53592901cf12b7e8b8881682f7484 unintentionally merged it back in.
I think we can just remove it here?

Reply:
wouldn't it make more sense to fix master and re-merge?

Reply:
I guess, maybe. Don't think it matters much, though.